packages/now-static-build/src/frameworks.ts

```diff
@@ -290,6 +290,11 @@ export const frameworks: Framework[] = [
       },
     ],
   },
+  {
+    name: 'Stencil',
+    dependency: '@stencil/core',
+    getOutputDirName: async () => 'www',
```

Comment:
Can you provide a link to the docs that show the output is www? I've never heard of Stencil? 🤓

Reply:
Ok I confirmed that `www` is the output directory.

However, I think you need to add routes similar to Svelte and UmiJS because refreshing the page doesn't work.

Also please add a test with several probes to confirm that your routes work properly.